test/vue-frag.spec.ts

```diff
@@ -1231,3 +1231,47 @@ test('Set innerHTML of empty fragment', async () => {
 	await wrapper.setData({ html });
 	expect(wrapper.html()).toBe(output);
 });
+
+test('keep-alive', async () => {
+	const usage = {
+		template: `
+			<app>
+				<frag v-frag>
```

Comment:
Don't need v-frag on frag

Reply:
@privatenumber
Sorry, but I'd have to disagree, almost all tests above are using v-frag on frag, otherwise they would fail

Reply:
Sorry my bad, I thought this was using the Frag component.